### DISCUSSION TITLE
error about Getting Started : Install MOOSE Conda Packages

### DISCUSSION THREAD

**Discussion by xiaojbing** [idaholab/moose#18625] [Q&A Getting Started] [answered]
I try to install moose in Ubuntu , but an error occurred during the installation of MOOSE Conda Packages.
![QQ截图20210818101326](https://user-images.githubusercontent.com/87591264/129825933-6193ab6a-e809-45e8-beb5-701fd008d333.png)

**Comment by GiudGiud** [comment]
Hello

Looks like a conda error to me. Not much we may be able to do about it. 
Can you try this instead
```conda deactivate
conda env remove -n moose
conda install mamba
mamba create -n moose moose-libmesh moose-tools
conda activate moose
```

Guillaume

**Reply by xiaojbing** [reply]
I followed your steps, but the following problems occurred
![2](https://user-images.githubusercontent.com/87591264/129826988-9e462eaa-cf8d-4522-8874-6861d4b75098.png)

**Reply by GiudGiud** [reply]
ok run conda deactivate first, then the other steps

**Reply by GiudGiud** [reply]
if this doesnt work, I'd remove conda completely and re-install

**Reply by cticenhour** [reply]
I would not uninstall conda, as it seems to be a channel issue. Please see my answer in this thread.

**Comment by cticenhour** [comment, marked as answer]
I just experienced this with a fresh conda installation, and it seems to be an issue with `https://conda.software.inl.gov`, as switching to `idaholab` allowed package metadata collection and package resolution. Tagging @milljm. 

In the meantime @xiaojbing, please run the following commands individually: 
```
conda deactivate
conda env remove -n moose
conda config --remove channels https://conda.software.inl.gov
conda config --add channels idaholab
conda create --name moose -q -y
conda activate moose
conda install moose-tools
conda install moose-libmesh
```
Now you should be able to attempt to build MOOSE. Do not forget to run `conda activate moose` anytime you start a new terminal instance.

If you run into issues, please copy and paste your error into the conversation rather than post screenshots! It helps others who may be encountering the same errors and searching the Discussions forum.

**Reply by xiaojbing** [reply]
Thank you, I ran the command and the problem has been solved.

**Reply by eastglow-zz** [reply]
I got the same issue with @xiaojbing on Big Sur, but it was resolved by this answer.

**Reply by milljm** [reply]
The channel is incorrect... Sorry. It should be:

```bash
conda config --add channels https://conda.software.inl.gov/public
```

I have a fix rolling through the system now: https://github.com/idaholab/moose/pull/18630

**Reply by eduardogrodriguez89** [reply]
I have this error and I coudn't solucionate it.

(moose) erodriguez@moccai-11:~$ conda install moose-libmesh
Collecting package metadata (current_repodata.json): failed

UnavailableInvalidChannel: The channel is not accessible or is invalid.
  channel name: idaholab
  channel url: https://conda.anaconda.org/idaholab
  error code: 404

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.

**Reply by cticenhour** [reply]
The idaholab conda channel has been closed due to Anaconda's Terms of Service. We are using the `https://conda.software.inl.gov/public` channel as our primary channel. Please run the following commands to fix your channels and to make sure your environment is set up properly to use the new channel:
```
conda deactivate
conda env remove -n moose
conda config --remove channels idaholab
conda config --add channels https://conda.software.inl.gov/public
conda create --name moose -q -y
conda activate moose
conda install moose-tools
conda install moose-libmesh
```

**Reply by eduardogrodriguez89** [reply]
thanks, it works
